grammar/artikel/2015-08-11-Bevölkerungswachstum-Gefahr-oder-Chance.md

```diff
@@ -0,0 +1,37 @@
+# Bevölkerungswachstum: Gefahr oder Chance?
+
+http://www.dw.com/de/bev%C3%B6lkerungswachstum-gefahr-oder-chance/a-18638722
+
+hallo Bekannte,
+
+ich las gestern einen sehr interessanten Artikel im Internet und ich empfehle,
+dass du auch diesen Artikel lesen und nachdenken. Die URL von diesem Artikel
+steht am Ende meinen Brief. Zuerst möchte ich dir meine kurze Zusammenfassung
+geben.
+
+Der Text geht um die Bevölkerung der Erde im Jahr 2100 und ob wir so viel
+Menschen auf der Erde unterstützen können. Eine Studie von der Vereinten
```

Comment:
unterstuetzen is more like "support". I would expect "so viele Menschen auf der Erde leben koennen"